### PR TITLE
Ngram api

### PR DESCRIPTION
**Pull request by pinggyu**
- working google ngram api call
- setup styling for ngram page and user handling input functions
- just need to link up with chart js to visualize

### CODE REVIEW

File: src/pages/NgramViewer.js [hunk 1]
```diff
@@ -1,43 +1,98 @@
...
-  // const getNgram = async () => {
-  //   await axios
-  //     .get('https://books.google.com/ngrams/json?content=Churchill&year_start=1800&year_end=2000&corpus=26&smoothing=3')
-  //     .then((response) => console.log(response))
-  //     .catch((err) => console.log(err));
-  // };
+  const getNgram = async () => {
+    await axios
+      .get(
+        `https://intense-dusk-96795.herokuapp.com/https://books.google.com/ngrams/json?content=${searchInput}&year_start=1800&year_end=2000&corpus=26&smoothing=3`
```
Comment (chxwcodes) [inline src/pages/NgramViewer.js:57]: yay for proxy servers 👏

File: src/pages/NgramViewer.js [hunk 2]
```diff
@@ -46,13 +101,68 @@ function NgramViewer() {
                 <a href="https://en.wikipedia.org/wiki/N-gram">Wikipedia</a>
```
Comment (chxwcodes) [inline src/pages/NgramViewer.js]: you can add a `target="_blank" rel="noopener noreferrer"` here so it opens to a new tab. If users comes back to our site they'll have to relogin

File: src/pages/NgramViewer.js [hunk 3]
```diff
@@ -46,13 +101,68 @@ function NgramViewer() {
...
+                    onChange={handleInput}
+                    placeholder="Frankenstein, Dracula..."
+                    // value={currentInput}
+                  />
+                </fieldset>
+
+                {/* If user wants to select from our list of saved backronyms */}
+                <fieldset>
+                  <legend>
+                    Or pick from our existing backronyms: (psst, you can add a new one <Link to="/">here</Link>)
```
Comment (chxwcodes) [inline src/pages/NgramViewer.js:145]: ❤️

**Review by chxwcodes — CHANGES_REQUESTED** [review]
lewks gooood
can add an open to new tag attribute to the a tags n we gucci

File: src/pages/NgramViewer.js (hunk 2 above)
Comment (pinggyu) [reply src/pages/NgramViewer.js]: truuuuu

Comment (pinggyu) [reply src/pages/NgramViewer.js]: THANK U!!

Comment (pinggyu) [reply src/pages/NgramViewer.js]: (added)

**Pull request merged**